spec/integration/local_sources/before/Podfile

```diff
@@ -4,6 +4,8 @@ ENV['COCOAPODS_DISABLE_STATS'] = '1'
 
 source 'https://github.com/CocoaPods/Specs.git'
 
+use_frameworks!
```

Comment:
this reduces the huge footprint of this PR since now the integration specs read the `Podfile` value and completely change to use libraries since the `Podfile` never had this set.